File: apps/chat-android/messaging-websocket/src/main/java/com/amazonaws/services/chime/sdk/messaging/session/ChimeUserCredentials.kt
/*
 * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 * SPDX-License-Identifier: MIT-0
 */

package com.amazonaws.services.chime.sdk.messaging.session

import java.io.Serializable

data class ChimeUserCredentials(
    val accessKeyId: String,
    val secretAccessKey: String,
    val sessionToken: String? = null
) : Serializable

File: apps/chat-ios/AmazonChimeSDKMessagingDemo/messaging-websocket/session/ChimeUserCredentials.swift
//
//  ChimeUserCredentials.swift
//  AmazonChimeSDKMessagingDemo
//
//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
//  SPDX-License-Identifier: MIT-0
//

import Foundation

struct ChimeUserCredentials: Codable {
    let accessKeyId: String
    let secretAccessKey: String
    let sessionToken: String
    
    enum CodingKeys: String, CodingKey {
        case accessKeyId = "AccessKeyId"
        case secretAccessKey = "SecretAccessKey"
        case sessionToken = "SessionToken"
    }
}
